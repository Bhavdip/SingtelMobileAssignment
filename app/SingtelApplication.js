import React, { Component } from 'react';
import { StyleSheet, View } from 'react-native';
import CardGame from './feature/card-game';

class SingtelApplication extends Component {
  render() {
    return (
      <View style={styles.sectionContainer}>
        <CardGame />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  sectionContainer: {
    flex: 1
  }
});
export default SingtelApplication;
