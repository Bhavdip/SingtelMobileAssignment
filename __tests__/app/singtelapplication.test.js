import React from 'react';
import { shallow } from 'enzyme';
import SingtelApplication from '../../app/SingtelApplication';

const setUpComponent = (props = {}) => {
  return shallow(<SingtelApplication {...props} />);
};

describe('Test SingtelApplication', () => {
  let wrapperComponent = {};
  let props = {};
  beforeEach(()=> {
    wrapperComponent = setUpComponent(props);
  });
  it('Test Snapshot Match', ()=> {
    expect(wrapperComponent).toMatchSnapshot();
  });
});
