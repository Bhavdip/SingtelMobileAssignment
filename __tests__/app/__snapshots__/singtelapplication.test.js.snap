// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SingtelApplication Test Snapshot Match 1`] = `ShallowWrapper {}`;
